3-enrich/scala-common-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/registry/AnonIpEnrichmentSpec.scala

```diff
@@ -23,20 +23,35 @@ import org.specs2.matcher.DataTables
  */
 class AnonIpEnrichmentSpec extends Specification with DataTables {
 
-  def is = s2"Anonymizing 0-4 octets across a variety of IP addresses should work $e1"
+  def is = s2"Anonymizing across a variety of IP addresses (v4 and v6) should work $e1"
```

Comment:
Can we come up with a property based test?

E.g. "Given valid ip address and `Octet.N` the final value always contains N `x`s"

Reply:
Or "Givena valid IP address and Octet - if we replace all `x`s with `1` it should be again a valid IP address"

Reply:
done